examples/apfl_example/server.py

```diff
@@ -22,10 +22,22 @@ def get_initial_model_parameters() -> Parameters:
     return ndarrays_to_parameters([val.cpu().numpy() for _, val in initial_model.state_dict().items()])
 
 
-def fit_config(local_epochs: int, batch_size: int, n_server_rounds: int, current_round: int) -> Config:
+def fit_config(
+    batch_size: int,
+    n_server_rounds: int,
+    current_round: int,
+    local_epochs: Optional[int] = None,
+    local_steps: Optional[int] = None,
+) -> Config:
+    if local_epochs is not None:
+        epochs_or_steps = {"local_epochs": local_epochs}
+    elif local_steps is not None:
+        epochs_or_steps = {"local_steps": local_steps}
+    else:
+        epochs_or_steps = {}
```

Comment:
very minor note: The server needs to provide one of local_epochs and local_steps or there will be a ValueError raised by the client. It might be a little confusing in the example to have it seem like providing neither is a valid option based on this code in the server but will throw an error on the client side.